src/PrestaShopBundle/Resources/views/Admin/Product/Include/form_combination.html.twig

```diff
@@ -1,3 +1,4 @@
+{% set is_stock_management = 'PS_STOCK_MANAGEMENT'|configuration %}
```

Comment:
Good idea thanks.
I will set a global twig variable named "configuration" to get all configurations not only PS_STOCK_MANAGEMENT.